README.md

```diff
@@ -164,9 +163,45 @@ For example:
                 fieldRef:
                   fieldPath: spec.nodeName
             - name: UPDATE_WINDOW_START
-              value: "9:0:0"
+              value: "09:00:00"
             - name: UPDATE_WINDOW_STOP
-              value: "21:0:0"
+              value: "21:00:00"
+```
+
+#### Set scheduler
+`SCHEDULER_CRON_EXPRESSION` can be used to specify the scheduler in which updates are permitted.
+When `SCHEDULER_CRON_EXPRESSION` is "* * * * * * *" (default), the feature is disabled.
+
+To enable this feature, go to `bottlerocket-update-operator.yaml` and change `SCHEDULER_CRON_EXPRESSION` to a valid cron expression.
```

Comment:
This instruction will change once helm is merged?

Reply:
Yeah, I think so. Maybe @jpmcb has more context on this.

Reply:
Yes - I'm planning to rebase all this on my PR once this gets in 😄